src/app.js

```diff
@@ -0,0 +1,7 @@
+import * as data from '../config/data.json';
+import * as models from './models';
+
+console.log(data.name);
+
+new models.User();
```

Comment:
```
import { User, Product } from './models';
```
Why not?
In this way we can make influence what a bundle will contain.

Reply:
Sounds good.
Done!